code/game/jobs/access.dm

```diff
@@ -138,25 +140,27 @@
 		ACCESS_PRESS,
 	)
 
-/proc/get_all_centcom_access()
+/proc/get_weyland_access(job)//Fairly sure this isn't really needed anymore
+	return get_all_weyland_access()
```

Comment:
The argument `job` is not used. Also might as well mention this as a code-doc:
```suggestion
///Currently no different than get_all_weyland_access
/proc/get_weyland_access()
	return get_all_weyland_access()
```

Reply:
Removed and replaced where it was used with get_all_weyland_access() as it did the same thing.